sky/templates/kubernetes-port-forward-proxy-command.yaml.j2

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Can you run `pytest tests/test_smoke.py --kubernetes -k "not TestStorageWithCredentials"` to make sure everything, including file_mounts, work correctly? I have manually verified, but going forward we want to run Kubernetes smoke tests for k8s PRs :)

Reply:
@romilbhardwaj Currently, passing all the tests besides the ones requiring GPUs for this branch.